operations/network/management_network/bican_technical_details.md

```diff
@@ -0,0 +1,286 @@
+
+In CSM 1.2, there will be Layer 3 separation internal to the system but co-mingled Layer 2 between the CHN IPv4 addressing and the internal HSN private IPv4 addresses.
+Isolation will be within the Slingshot network as well as separated at the Edge Router.
+
+
+#### 2.5.1 Touchpoints: effects and changes
+
+- Edge Router provides all **routing and access controls** in CSM 1.2 (via a virtual routing instance if Arista switch pair is used).
+As noted above, non-Arista router configurations are out of scope for CSM 1.2 work.
+- Internal to the system CHN traffic will exist in the same Layer 2 domain with internal HSN traffic until the Slingshot network supports VLAN separation.
+- Compute (CN) and Application Node (UAN in this case) configuration or IPv4 addressing and routing will be via CFS.
+  - When multiple HSN interfaces exists the CHN can be configured to load balance TCP/UDP traffic &quot;flows&quot; across interfaces via ECMP Layer 3 routing in Linux.
+- UAI addressing and routing over the HSN interfaces for the NCN workers is required.
+- API endpoints in MetalLB for the CHN will be accessible over NCN worker HSN interfaces (via ECMP Layer 3 routing).
+  - MetalLB will peer with the Edge Routers to supply load balanced API access.
+
+
+#### 2.5.2 When configuration occurs
+
+- Installation as part of a virtual routing instance on the Edge Routers.
+
+
+#### 2.5.3 Ability to change post install
+
+- NOT RECOMMENDED - Edge Router controls external access so change scope is limited.
+- NO - Node images could be changed but routing and IP changes to CFS images would need extensive testing to certify.
+
+
+## 3 Management CAN(CMN)
+
+The original CAN released in Shasta 1.1 contained the ability to access NCN workers, masters and storage directly via SSH for administrative purposes.
+This administrative traffic was co-mingled with general user traffic for jobs.
+Based on customer requests a new mechanism for administrative access to workers, masters and storage nodes will be added in CSM 1.2.
+The new Customer Management Network (CMN) will be created as **a separate-and-distinct VLAN and Subnet on the Management Network** and uplink at the edge to the Customer network.
+This new CMN network will allow SSH into the NCNs and CAN access will be disallowed.
+NOTE this is generally for ingress access for administrative purposes.
+
+
+### 3.1 Traffic Separation and Routing
+
+Enabling the CMN at installation time will have the following effects:
+
+- Adding uplinks to the Customer Site similar to the original CAN.
+- Creating of a new VLAN on the Management Network.
+- Adding a new subnet and routing to the Management Network and NCNs.
+
+
+### 3.2 Endpoint Naming
+
+If enabled during installation a ".CMN" suffix will be generated by CSI and used in SLS and Authoritative DNS.
+The "plumbing" of this will occur as previously described in the CHN.
+
+Examples:
+
+- SSH `ncn-w001.cmn.tld`
+
+<a name="3-3-endpoint-addressing"></a>
```

Comment:
These anchors aren't necessary, please remove here and in other pages that have them added.

Reply:
try #2 to remove all anchor references